documentation/modules/auxiliary/dos/http/slow_loris.md

```diff
@@ -0,0 +1,43 @@
+## Vulnerable Application
+
+This module tries to keep many connections to the target web server open and hold them open as long as possible.
+
+Vulnerable app versions include:
+
+- Apache HTTP Server 1.x and 2.x
+- Apache Tomcat 5.5.0 through 5.5.29, 6.0.0 through 6.0.27 and 7.0.0 beta
+
+Download the Metasploitable 2 vulnerable Linux virtual machine from [https://sourceforge.net/projects/metasploitable/files/Metasploitable2/](https://sourceforge.net/projects/metasploitable/files/Metasploitable2/).
+
+## Verification Steps
+
+1. Start msfconsole
+2. Do: use auxiliary/dos/http/slow_loris
+3. Do: set RHOST
+4. Do: run
+5. Visit server URL in your web-browser.
+
```

Comment:
Add a step that tells users what the expected results are from running the module.